### PR TITLE
[fabric] Changes to support external chaincode

### PR DESCRIPTION
**Pull request by sownak**
- Changes to support TLS external chaincode
- Renamed chaincode containers so that Flux does not give releaseName format errors because of long names
- removed references to hyperledgerlabs docker images
- updated sstorgacelass name

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/roles/create/chaincode/peer_certs/tasks/generate_certs.yaml [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
-  register: vault_result
+  register: client_cert_result
   ignore_errors: yes
 
-# This task clean up variable for the next org
-- name: Clean up variable
-  set_fact:
-    cert_generated: !!null
-  when: vault_result.failed == True
+# # This task clean up variable for the next org
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-fabric/configuration/roles/create/chaincode/peer_certs/tasks/generate_certs.yaml:18]: Is it commented out temporarily or can be removed ?

File: platforms/hyperledger-fabric/charts/external_chaincode/templates/deployment.yaml [hunk 2]
```diff
@@ -111,8 +111,8 @@ spec:
     {{ end  }}
       containers:
         - image: {{ $.Values.metadata.images.external_chaincode }}
-          name: "{{ $.Values.chaincode.name }}-{{ $.Values.chaincode.version }}-{{ $.Values.chaincode.org }}"
-          imagePullPolicy: IfNotPresent
+          name: "{{ $.Values.chaincode.name }}-{{ $.Values.chaincode.version }}"
+          imagePullPolicy: Always
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-fabric/charts/external_chaincode/templates/deployment.yaml:115]: Any reason for pull policy to be always ?

**Review by suvajit-sarkar — COMMENTED** [review]
Reset LGTM

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM

**Pull request merged**